experimental/Schemes/SpaceGerms.jl

```diff
@@ -296,8 +299,8 @@ julia> defining_ring_element(XS)
 
 ```
 """
-defining_ring_element(X::HypersurfaceGerm) = X.f
-defining_ring_elements(X::CompleteIntersectionGerm) = X.v
+defining_ring_element(X::HypersurfaceGerm) = X.f::MPolyLocRingElem
+defining_ring_elements(X::CompleteIntersectionGerm) = X.v::Vector{MPolyLocRingElem}
```

Comment:
These types are still not concrete, so unfortunately it doesn't really help to produce type stability. It should probably be `elem_type(localized_ring_type(ring_type(X)))`.

Reply:
Thank you. I will try that.

Reply:
Given the latest failures in the tests, I suggest to not do a type assertion in the second case, but a conversion with 
`Vector{elem_type(...)}(X.v)`. Or you do that conversion in the constructor(s) and we hope that it will prevail.

Reply:
Thank you for your help in the commit below resolving this problem correctly.